Assets/Mirror/Examples/MultipleAdditiveScenes/Scripts/PhysicsCollision.cs

```diff
@@ -0,0 +1,44 @@
+﻿using UnityEngine;
+
+namespace Mirror.Examples.MultipleAdditiveScenes
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public class PhysicsCollision : NetworkBehaviour
+    {
+        [Tooltip("how forcefully to push this object")]
+        public float force = 12;
+
+        public Rigidbody rigidbody3D;
+
+        private void OnValidate()
+        {
+            if (rigidbody3D == null)
+                rigidbody3D = GetComponent<Rigidbody>();
+        }
+
+        private void Start()
+        {
+            rigidbody3D.isKinematic = !isServer;
+        }
+
+        [ServerCallback]
+        void OnCollisionStay(Collision c)
+        {
+            if (c.gameObject.CompareTag("Player"))
+            {
+                // get direction from which player is contacting object
+                Vector3 dir = c.contacts[0].normal;
+
+                // zero the y and normalize so we don't shove this through the floor or launch this over the wall
+                dir.y = 0;
+                dir = dir.normalized;
+
+                // push this away from player...a bit less force for host player
+                if (c.gameObject.GetComponent<NetworkIdentity>().connectionToClient.connectionId == 0)
```

Comment:
`c.gameObject.GetComponent<NetworkIdentity>().connectionToClient.connectionId == 0`

maybe put this into its own function, something like `IsHostPlayer(GameObjcet)`

Reply:
Why make a separate method for 1 line of code that is only called in one place?  That's getting a bit extreme.

Reply:
If some has to read the comment then the line isnt clear enough. Its not extreme its the same reason for good variable names

Reply:
Better idea - give NetworkIdentity an `isHost` bool I can check.